### PR TITLE
Set wasmvm cache size to zero.

### PR DESCRIPTION
**Pull request by DimitrisJim**
## Description

Another fix from SImon's review

closes: #XXXX

### Commit Message / Changelog Entry

```text
chore: set wasm cache size to zero.
```

see the [guidelines](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) for commit messages. (view raw markdown for examples)

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#pull-request-targeting)).
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/build/building-modules/11-structure.md) and [Go style guide](../docs/dev/go-style-guide.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/main/testing/README.md#ibc-testing-package).
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`).
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Provide a [commit message](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) to be used for the changelog entry in the PR description for review.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.
- [ ] Review `Codecov Report` in the comment section below once CI passes.

### CODE REVIEW

**Review by chatton — APPROVED** [review]
LGTM thanks for the fix

File: modules/light-clients/08-wasm/types/config.go [hunk 1]
```diff
@@ -6,10 +6,11 @@ const (
 	// contractMemoryLimit is the memory limit of each contract execution (in MiB)
 	// constant value so all nodes run with the same limit.
 	ContractMemoryLimit = 32
+	// The wasm vm cache size is set to 0 to reduce unnecessary memory usage. See: https://github.com/CosmWasm/cosmwasm/pull/1925
+	MemoryCacheSize = 0
```
Comment (damiannolan) [inline modules/light-clients/08-wasm/types/config.go:10]: nit: godocs for exported symbols should start with the name of the symbol 

(maybe if you decide to fix this you could fix above `ContractMemoryLimit`'s godoc is lower camel case when it should be pascal case

**Review by damiannolan — APPROVED** [review]
Looks good to me

File: modules/light-clients/08-wasm/types/config.go (hunk 1 above)
Comment (DimitrisJim) [reply modules/light-clients/08-wasm/types/config.go:11]: ah, yeah. Gladly fixing both.

**Review by crodriguezvega — APPROVED** [review]

**Review by charleenfei — APPROVED** [review]

**Pull request merged**